docs/intro/install.rst

```diff
@@ -265,7 +265,7 @@ For details, see `Issue #2473 <https://github.com/scrapy/scrapy/issues/2473>`_.
 .. _cryptography: https://cryptography.io/en/latest/
 .. _pyOpenSSL: https://pypi.org/project/pyOpenSSL/
 .. _setuptools: https://pypi.python.org/pypi/setuptools
-.. _AUR Scrapy package: https://aur.archlinux.org/packages/scrapy/
+.. _ArchlinuxScrapyPackage: https://www.archlinux.org/packages/community-staging/any/scrapy/
```

Comment:
Changing this name without changing its occurrences in the text looked strange to me, so I checked and it seems like there are in fact no occurrences at all. Should we remove it?

Reply:
I agree it doesn't appear to be referenced anywhere. I think we can remove it. Should I make the change?

Reply:
Yes, please do. Thanks.